packages/squiggle-lang/src/rescript/ReducerProject/ReducerProject_ProjectItem.res

```diff
@@ -182,7 +182,6 @@ let doRun = (this: t, context: Reducer_T.context): t =>
         ->setContinuation(contextAfterEvaluation.bindings->Reducer_Bindings.locals)
       } catch {
       | Reducer_ErrorValue.ErrorException(e) => this->failRun(e)
-      | _ => this->failRun(RETodo("unhandled rescript exception"))
```

Comment:
Why did you delete this?

Reply:
Woops! This shouldn't be here. For some reason, runProject still doesn't run my GiveDirectly example. I wanted to see the real error by removing this line.